packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -20,6 +21,156 @@ import 'scroll_position.dart';
+/// efficient when there are too many or infinite children, by bringing in
+/// only the children that are visible in viewport, and removing them when they
+/// are no longer visible.
+///
+/// See also:
+///  * [ListWheelChildListDelegate], a delegate that supplies children using an
+///    explicit list.
+///  * [ListWheelChildLoopingListDelegate], a delegate that supplies infinite
+///    children by looping an explicit list.
+///  * [ListWheelChildBuilderDelegate], a delegate that supplies children using
+///    a builder callback.
+abstract class ListWheelChildDelegate {
+  /// Return the child at the given index. If the child at the given
+  /// index does not exist, return null.
+  Widget build(BuildContext context, int index);
+
+  /// Returns an estimate of the number of children this delegate will build.
+  int get estimatedChildCount;
+
+  /// Returns the true index for a child built at a given index. Defaults to
+  /// the given index, however if the delegate is [ListWheelChildLoopingListDelegate],
+  /// this value is the index of the true element that the delegate is looping to.
+  int trueIndexOf(int index) => index;
+
+  /// Called to check whether this and the old delegate are actually 'different',
+  /// so that the caller can decide to rebuild or not.
+  bool shouldRebuild(covariant ListWheelChildDelegate oldDelegate);
+}
+
+/// Class that supplies children for the [ListWheelScrollView].
+///
+/// Instead of providing a concrete list of children to [ListWheelScrollView],
+/// this class lazily supplies children using an explicit list. This is more
+/// efficient when there are too many children in the list, by bringing in
+/// only the children that are visible in viewport, and removing them when they
+/// are no longer visible.
+///
+/// Children are available in the range [0, explicit_list.length - 1], outside
+/// of that range a null child will be returned.
+class ListWheelChildListDelegate extends ListWheelChildDelegate {
+  /// Constructs the delegate from a concrete list of children.
+  ListWheelChildListDelegate({@required this.children}) : assert(children != null);
+
+  /// The list containing all children that can be supply.
+  final List<Widget> children;
+
+  @override
+  int get estimatedChildCount => children.length;
+
+  @override
+  Widget build(BuildContext context, int index) {
+    if (index < 0 || index >= children.length)
+      return null;
+    return children[index];
+  }
+
+  @override
+  bool shouldRebuild(covariant ListWheelChildListDelegate oldDelegate) {
+    return children != oldDelegate.children;
+  }
+}
+
+
+/// Class that supplies children for the [ListWheelScrollView].
+///
+/// Instead of providing a concrete list of children to [ListWheelScrollView],
+/// this class lazily supplies an infinite amount of children from an explicit
+/// list by looping the list. The class only brings in children that are visible
+/// in the viewport, and removes them when they are no longer visible.
```

Comment:
Be clear here and above that by 'The class', you're describing that ListWheelScrollView's render object and element removes children when they're no longer visible. This delegate can't remove children.